### ISSUE TITLE
add command line flags to julia installer on Windows

### ISSUE THREAD

**Issue by stonebig** [JuliaLang/julia#10126]
Would it be possible to have a silent installer on windows ?
Ideal whish:
````
julia-0.3.6-win64.exe /D=%JULIAROOT% /OpenJulia=No /StartMenu=No /DesktopShortcut=No
````

reference:

 https://groups.google.com/forum/#!searchin/julia-users/installer/julia-users/tT_kGJwHISE/jAWSWPmNScAJ

**Comment by tkelman** [comment]
Did you try `/S` ? Not sure whether the shortcut options are scriptable. Might be.

**Comment by stonebig** [comment]
hi @tkelman ,  `/S` alone would not suffice, as I whish also to unclick the shortcut options.

**Comment by ihnorton** [comment]
This can be implemented (PRs welcome). As a work-around you can actually run `7z x` on the download binary, which will extract `julia-installer.exe`. Run `7z x julia-installer.exe` again to get a directory called $_OUTDIR which is equivalent to the install directory, minus the shortcut.

**Comment by FlorianRappl** [comment]
Hm is this still open? I just checked and applying `/S` on the compressed installer (downloaded from https://s3.amazonaws.com/julialang/bin/winnt/x64/0.3/julia-0.3.7-win64.exe) worked. So apparently it already calls `julia-installer.exe` with `/S` after extraction.

**Comment by stonebig** [comment]
It's open until it's `done` or `won't fix`.

**Comment by FlorianRappl** [comment]
Ah I see - so you want the whole package and `/S` alone was working all the time. That's good to know - I was actually misinterpreting the discussion then.

**Comment by ihnorton** [comment]
closing per comment in #13303:

Actually, it is possible to do a silent install with the current self-extracting binary:

`julia-0.5.0-da085b479c-win64 /S /D=C:\juliatest`

Two caveats:
- this launches a GUI process and returns to `cmd` immediately. I'm not sure how to wait on completion, but `Uninstaller.exe` should be the last thing written, so could wait for that.
- as far as I can tell, the `/D=` parameter *must* be a full path (or I may have a bunch of copies in temp dirs...)

**Comment by tkelman** [comment]
There was a feature request here for more flags to disable the shortcut generation but that's firmly in "pull requests welcome" category
